jwtauthenticator/jwtauthenticator.py

```diff
@@ -12,30 +12,51 @@ def get(self):
         header_name = self.authenticator.header_name
         auth_header_content = self.request.headers.get(header_name, "")
         signing_certificate = self.authenticator.signing_certificate
+        secret = self.authenticator.secret
         username_claim_field = self.authenticator.username_claim_field
         audience = self.authenticator.expected_audience
+        tokenParam = self.get_argument('token', default=False)
```

Comment:
I thought that too..  but https://jwt.io/introduction/ states you should use 'Authorization: Bearer  " which seems to me to indicate JWT does defer to OAuth?   OK, so in that case I propose we let the default be "access_token" for the query parameter as you proposed.  I'll try to push a change sometime today after I clear a couple of things from my plate.